lib/beaker/tasks/quick_start.rb

```diff
@@ -0,0 +1,49 @@
+require 'beaker-hostgenerator'
+
+namespace :beaker_quickstart do
+
+  desc 'Generate Default Beaker Host Config File'
+  task :gen_hosts do
```

Comment:
maybe generation tasks should have their own namespace, since they're separate from running beaker itself?